src/docs/reference/modules/Scripting/README.md

```diff
@@ -14,18 +14,31 @@ The main interface is `IScriptingManager`.
 public interface IScriptingManager
 {
     IScriptingEngine GetScriptingEngine(string prefix);
-    object Evaluate(string directive);  
-    IList<IGlobalMethodProvider> GlobalMethodProviders { get; }
```

Comment:
I think we agreed to reference the file and remove the code instead of update it each time when there's a change

Reply:
Done.